### PR TITLE
補埋GA 

### LINKED ISSUE #251: 補埋GA
- [x] about section 開始參與鍵底下的help-text
- [x] 每個頁面的navigation bar的link 

ref: #242

### PR DESCRIPTION
**Pull request by barry800414**
to fix #251

### CODE REVIEW

File: about_us.html [hunk 1]
```diff
@@ -151,6 +151,10 @@ <h3 class="text-center">加入我們</h3>
 
     <!-- font CSS -->
     <link href="css/font.css" rel="stylesheet">
+
+    <!-- Google analytics-->
+    <script src="js/general-i13n.js"></script>
+
```
Comment (mark86092) [inline about_us.html]: please remove the white space

File: js/general-i13n.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+$(function(){
+    // Tracking click events
+    $("a").click(function(e){
+        if (this.dataset['logcategory'] && this.dataset['logaction'] && ga) {
+            var category = this.dataset['logcategory'];
+            console.log(category);
```
Comment (mark86092) [inline js/general-i13n.js]: no `console.log`

File: about_us.html (hunk 1 above)
Comment (barry800414) [inline about_us.html]: Sorry, where do you mean?

Comment (mark86092) [inline about_us.html]: L157

**Pull request merged**